binary_info/blink_any/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+if (PICO_CYW43_SUPPORTED AND NOT TARGET pico_cyw43_arch)
+    return()
```

Comment:
why do some of these have a message (there is one without below too), and some not?

Reply:
So you don't see the same message repeatedly?